tests/integration/helpers.py

```diff
@@ -74,3 +119,77 @@ async def restart_application(ops_test: OpsTest):
         .run_action("restart")
     )
     await action.wait()
+
+
+async def get_access_token(ops_test: OpsTest, base_url):
+    """Get access token of the `admin` user.
+
+    Args:
+        ops_test: PyTest object.
+        base_url: Superset URL.
+
+    Returns:
+        headers: Request headers with access token.
+    """
+    response = requests.post(
+        base_url + "/api/v1/security/login", json=API_AUTH_PAYLOAD, timeout=30
+    )
+    access_token = response.json()
+    headers = {"Authorization": "Bearer " + access_token["access_token"]}
+    return headers
+
+
+async def get_chart_count(ops_test: OpsTest, url, headers):
+    """Count Superset charts.
+
+    Args:
+        ops_test: PyTest object.
+        url: Superset URL.
+        headers: Request headers with access token.
+
+    Returns:
+        Count of Superset charts.
+    """
+    chart_response = requests.get(
+        url + "/api/v1/chart/", headers=headers, timeout=30
+    )
+    charts = chart_response.json()
```

Comment:
Unsure, since it's an integration test, if there is an exception here I would like the test to fail, I'll add a try except which logs the error and then it can fail on the assert.